docs/middlewares.md

```diff
@@ -466,6 +467,58 @@ handler
   .use(s3KeyNormalizer())
 ```
 
+
+## [secretsManager](/src/middlewares/secretsManager.js)
+
+Fetches parameters from [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html).
+
+Secrets to fetch can be defined by by name. See AWS docs [here](https://docs.aws.amazon.com/secretsmanager/latest/userguide/tutorials_basic.html).
+
+Secrets are assigned to the function handler's `context` object.
+
+The Middleware makes a single [API request](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html) for each secret as Secrets Manager does not support batch get.
+
+For each secret, you also provide the name under which its value should be added to `context`.  
+
+### Options
+
+- `cache` (boolean) (optional): Defaults to `false`. Set it to `true` to skip further calls to AWS Secrets Manager
+- `cacheExpiryInMillis` (int) (optional): Defaults to `undefined`. Use this option to invalidate cached secrets from Secrets Manager
+- `secrets` (object) : Map of secrets to fetch from Secrets Manager, where the key is the destination, and value is secret name in Secrets Manager.
+  Example: `{secrets: {RDS_LOGIN: 'dev/rds_login'}}`
+- `awsSdkOptions` (object) (optional): Options to pass to AWS.SecretsManager class constructor.
+
+NOTES:
+* Lambda is required to have IAM permission for `secretsmanager:GetSecretValue` action
+* `aws-sdk` version of `2.176.0` or greater is required. If your project doesn't currently use `aws-sdk`, you may need to install it as a `devDependency` in order to run tests
+
+### Sample Usage
+
+Simplest usage, exports parameters as environment variables.
```

Comment:
>  exports parameters as environment variables.

Copy-paste from SSM middleware?) Looks like no option to export as environment variable in this middleware, only `context` object